api/envoy/api/v2/auth/cert.proto

```diff
@@ -391,13 +392,19 @@ message DownstreamTlsContext {
     // Config for fetching TLS session ticket keys via SDS API.
     SdsSecretConfig session_ticket_keys_sds_secret_config = 5;
   }
+
+  // If specified, session_timeout will change maximum lifetime (in seconds) of ssl session
```

Comment:
Can you link (with RST syntax) to the RFC or man pages that define what a session is? Also, tiniest nit, please capitalize SSL (or just use TLS for consistency, we don't have any plain SSL support in Envoy anymore).

Reply:
sure.